.github/workflows/ci.yml

```diff
@@ -13,6 +13,8 @@ jobs:
             tox-environment: py37-Plone52
           - python-version: 3.8
             tox-environment: py38-Plone52
+          - python-version: 3.9
+            tox-environment: py39-Plone52
```

Comment:
`Plone` 5.2 does not support Python 3.9. See:

https://github.com/plone/Products.CMFPlone/blob/c90d3e37f722eec4c004bb132f4427d77f32fa24/setup.py#L25-L28

So it's wrong to do this.

Can this PR just fix the buildout pinn?

Reply:
OK. I've seen some change logs about Python 3.9 comparability, but it was not Plone 5.2 core itself. My bad.